packages/connect/src/ExtensionProvider/ExtensionProvider.ts

```diff
@@ -321,6 +333,7 @@ export class ExtensionProvider implements ProviderInterface {
     this.#connectionStatePingerId = setInterval(this.#checkClientPeercount, 
       this.healthPingerInterval);
 
+    // this.#isConnected = true;
     return Promise.resolve();
```

Comment:
This shouldn't be commented out

Reply:
This actually should be removed - It is a stray-workaround for checking when there were no Peers - I will remove it